dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/datavalueset/SpringDataValueSetStore.java

```diff
@@ -153,10 +153,11 @@ private String buildDataValueSql( Date lastUpdated, IdSchemes idSchemes )
         String deScheme = idSchemes.getDataElementIdScheme().getIdentifiableString().toLowerCase();
         String ouScheme = idSchemes.getOrgUnitIdScheme().getIdentifiableString().toLowerCase();
         String ocScheme = idSchemes.getCategoryOptionComboIdScheme().getIdentifiableString().toLowerCase();
+        String aocScheme = idSchemes.getAttributeOptionComboIdScheme().getIdentifiableString().toLowerCase();
 
         final String sql = "select de." + deScheme + " as deid, pe.startdate as pestart, pt.name as ptname, ou."
             + ouScheme + " as ouid, " +
-            "coc." + ocScheme + " as cocid, aoc." + ocScheme + " as aocid, " +
+            "coc." + ocScheme + " as cocid, aoc." + aocScheme + " as aocid, " +
```

Comment:
I guess using `ocScheme` for AttributeOptionCombo as well was somewhat intentional. 
Maybe we should only use `getAttributeOptionComboIdScheme` when it is set and otherwise fallback on `getCategoryOptionComboIdScheme` ?